src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        // Read file and calculate totals
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
```

This violates checklist item: "Lines should be split by comma into exactly two parts; handle malformed lines by throwing a RuntimeException (as part of not ignoring errors)." You should check if parts.length != 2 and throw a RuntimeException with a meaningful message if so.